src/Admin/AbstractAdmin.php

```diff
@@ -1966,11 +1970,6 @@ public function getSecurityHandler(): ?SecurityHandlerInterface
         return $this->securityHandler;
     }
 
-    /**
-     * NEXT_MAJOR: Decide the type declaration for the $name argument, since it is
```

Comment:
What is the decision?

Reply:
`string|name` is used on the interface in 3.x (and master)